thread_win32.c

```diff
@@ -768,7 +767,7 @@ timer_thread_func(void *dummy)
     RUBY_DEBUG_LOG("start");
     rb_w32_set_thread_description(GetCurrentThread(), L"ruby-timer-thread");
     while (WaitForSingleObject(timer_thread.lock,
-                               TIME_QUANTUM_USEC/1000) == WAIT_TIMEOUT) {
+                               thread_default_quantum_ms) == WAIT_TIMEOUT) {
```

Comment:
I'm not 100% sure this is the same as the other purposes we've used the ENV var. This looks a bit more like the "minimum quantum" the pthread code uses on the poll (it is also 10ms).